packages/@sanity/desk-tool/src/panes/document/statusBar/sparkline/DocumentSparkline.tsx

```diff
@@ -1,74 +1,85 @@
-import {EditIcon} from '@sanity/icons'
 import {Box, Flex, useElementRect} from '@sanity/ui'
-import React, {useEffect, useMemo, useState, useRef, memo} from 'react'
-import {raf2} from '../../../../utils/raf'
+import React, {useEffect, useMemo, useState, memo, useLayoutEffect} from 'react'
+import {useSyncState} from '@sanity/react-hooks'
 import {useDocumentPane} from '../../useDocumentPane'
 import {DocumentBadges} from './DocumentBadges'
-import {ReviewChangesButton} from './ReviewChangesButton'
-import {IconBadge} from './IconBadge'
-import {
-  BadgesBox,
-  MetadataBox,
-  ReviewChangesBadgeBox,
-  ReviewChangesButtonBox,
-} from './DocumentSparkline.styled'
 import {PublishStatus} from './PublishStatus'
+import {ReviewChangesButton} from './ReviewChangesButton'
 
 export const DocumentSparkline = memo(function DocumentSparkline() {
-  const {editState, historyController, value} = useDocumentPane()
+  const {
+    editState,
+    historyController,
+    value,
+    documentId,
+    documentType,
+    handleHistoryClose,
+    handleHistoryOpen,
+    changesOpen,
```

Comment:
Very nitpick – I like to alphabetize these things to increase readability. The "Sort lines" plugin in VS code is nice for this.